core/templates/pages/contributor-dashboard-page/services/translate-text-backend-api.service.spec.ts

```diff
@@ -283,5 +301,24 @@ describe('TranslateTextBackendApiService', () => {
           'html')
       ).toBeRejectedWithError('No image data found');
     });
+
+    it('should throw error if prefix', async() => {
```

Comment:
Try to write this a bit more clearly -- surely the condition should be something like "if prefix is invalid"?

Reply:
Done.

Reply:
Done